optimism.tokenlist.json

```diff
@@ -1067,6 +1067,61 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 10,
+      "address": "0x3bb4445d30ac020a84c1b5a8a2c6248ebc9779d0",
+      "name": "Theranos Coin",
+      "symbol": "LIZ",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/LIZ.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 1,
+      "address": "0x42d6622dece394b54999fbd73d108123806f6a18",
+      "name": "SPANK",
+      "symbol": "SPANK",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/spank.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0xcfD1D50ce23C46D3Cf6407487B2F8934e96DC8f9",
+      "name": "SPANK",
+      "symbol": "SPANK",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/spank.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x2eA8f9B29AC1d70e01AE9D23d0451D2242e8d609",
```

Comment:
Verify this token in Etherscan

Reply:
fixed

Reply:
Can you give my account 0xe3a90926133fef3c58514fA1aAeF75E0998aedD5 some kovan tokens I can test with? This is all in the repo readme